<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCard</class>
 <widget class="QMainWindow" name="AddCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);




</string>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>30</y>
      <width>1311</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(217, 197, 126);
border: none</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>-1</y>
       <width>1281</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Add Cards</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>240</y>
      <width>411</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar::Tab {
	width: 200;
	height: 40;
	border: 1px solid black;
	border-radius: 4;
	color: black;

}
QTabWidget::Pane {
	background-color: #fafafa;
}
QTabBar::Tab::Selected{
	background-color: #d9c57e;


}
QLineEdit {
	background-color: none;
	border: 1px solid black;
 	border-radius: 4;

}

QPushButton {
	border: 1px solid black;
	background-color: #d9c57e;
	border-radius: 4;


}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Single Card</string>
     </attribute>
     <widget class="QFrame" name="cardImage">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>50</y>
        <width>171</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid black;
border-radius: 5</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QComboBox" name="numberCombo">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>170</y>
        <width>203</width>
        <height>33</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Card Number</string>
      </property>
     </widget>
     <widget class="QComboBox" name="setCombo">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>110</y>
        <width>203</width>
        <height>33</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Set</string>
      </property>
     </widget>
     <widget class="QComboBox" name="franchiseCombo">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>50</y>
        <width>203</width>
        <height>33</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 5;
background-color: rgb(255, 255, 255);
border: 1px solid black;
 QComboBox::down-arrow {image: url(noimg); border-width: 0px;}</string>
      </property>
      <property name="currentText">
       <string/>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="maxVisibleItems">
       <number>10</number>
      </property>
      <property name="maxCount">
       <number>2147483646</number>
      </property>
      <property name="placeholderText">
       <string>Franchise</string>
      </property>
      <item>
       <property name="text">
        <string>Pokemon</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Yu-Gi-Oh!</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Multiple Cards</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>512</y>
      <width>204</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black;
border-radius: 5;
color: black;
background-color: rgb(217, 197, 126);

QPushButton::hover {
	
	background-color: rgb(255, 85, 255);
}</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
